msg/messages.js

```diff
@@ -24,242 +24,256 @@
  *
  * After modifying this file, run `npm run translate` from the root directory
  * to regenerate `./msg/json/en.json`.
+ * IMPORTANT:
+ * All message strings must use double quotes for the scripts to work properly
```

Comment:
This comment should be updated now that the strings have changed to use single quotes instead of double quotes.